applications/CompressiblePotentialFlowApplication/CMakeLists.txt

```diff
@@ -2,44 +2,69 @@ set(CMAKE_INCLUDE_CURRENT_DIR ON)
 
 message("**** configuring KratosCompressiblePotentialFlowApplication ****")
 
+################### PYBIND11
+include(pybind11Tools)
+
 include_directories( ${CMAKE_SOURCE_DIR}/kratos )
+include_directories( ${CMAKE_SOURCE_DIR}/applications/FluidDynamicsApplication )
```

Comment:
What happens if Fluid is not compiled?
=> How abt throwing a proper error?
Sth like [this](https://github.com/KratosMultiphysics/Kratos/blob/master/applications/IgaApplication/CMakeLists.txt#L8-L15)

Reply:
Maybe it makes more sense to switch on the FluidDynamicsApplication compilation flag instead of throwing the error. What do you think?

Reply:
I think @jcotela told us last time that this is not possible as of now

Reply:
Forget about it. This implies the fill the compilation files with if statements.

Reply:
I think it is possible but using a quite dirty solution.

Reply:
Done. If FluidDynamicsApplication is switched of the error is raised.